doc/api/path.md

```diff
@@ -330,6 +330,15 @@ For example on POSIX:
 path.normalize('/foo/bar//baz/asdf/quux/..');
 // Returns: '/foo/bar/baz/asdf'
 ```
+*Note*: The `path.posix.normalize()` method will not attempt to convert `\ ` 
+(Windows) to `/` (POSIX), as `\ ` is not recognized by POSIX as a valid 
+directory separator.  
+
+For example:
+```js
+path.posix.normalize("\\..\\some\\thing\\like\\this")
+//Returns '\..\some\thing\like\this'
```

Comment:
It does not return that string:
```
> path.posix.normalize("\\..\\some\\thing\\like\\this")
'\\..\\some\\thing\\like\\this'
```
Either remove the string quotes, or leave the string quotes and properly escape the backslashes (I suggest the latter)